commands.js

```diff
@@ -722,6 +722,7 @@ exports.commands = {
 
 		room.auth[targetUser.userid] = '#';
 		this.addModCommand("" + name + " was appointed Room Owner by " + user.name + ".");
+		targetUser.popup("You were appointed Room Owner by " + user.name + " in " + room + ".");
```

Comment:
<code>room</code> is an object in these commands so you should use <code>room.id</code> instead.